editors/code/src/client.ts

```diff
@@ -5,6 +5,30 @@ import { Config } from './config';
 import { CallHierarchyFeature } from 'vscode-languageclient/lib/callHierarchy.proposed';
 import { SemanticTokensFeature, DocumentSemanticsTokensSignature } from 'vscode-languageclient/lib/semanticTokens.proposed';
 
+export function configToServerOptions(config: Config): object {
```

Comment:
```suggestion
export function configToServerOptions(config: Config) {
```
I'd suggest not doing this upcast here, let the type inference make its job...